Parse/src/main/java/com/parse/ParseQuery.java

```diff
@@ -297,7 +297,7 @@ private static void throwIfLDSEnabled(boolean enabled) {
     }
   }
 
-  /* package */ static class State<T extends ParseObject> {
+  /* package */ public static class State<T extends ParseObject> {
```

Comment:
@rogerhu Can we remove the `/* package */ lines since they would be irrelevant since the class is public?